src/systemd/sd-dhcp6-client.h

```diff
@@ -85,8 +85,9 @@ int sd_dhcp6_client_set_index(sd_dhcp6_client *client, int interface_index);
 int sd_dhcp6_client_set_local_address(sd_dhcp6_client *client, const struct in6_addr *local_address);
 int sd_dhcp6_client_set_mac(sd_dhcp6_client *client, const uint8_t *addr,
                             size_t addr_len, uint16_t arp_type);
-int sd_dhcp6_client_set_duid(sd_dhcp6_client *client, uint16_t type, uint8_t *duid,
-                             size_t duid_len);
+int sd_dhcp6_client_set_duid(sd_dhcp6_client *client, uint16_t duid_type,
+                             size_t duid_len, uint8_t *duid);
+int sd_dhcp6_client_set_iaid(sd_dhcp6_client *client, uint32_t iaid);
```

Comment:
sd_dhcp6_client_set_iaid is a new API needed to configure IAID. I reversed the parameter order change in sd_dhcp6_client_set_duid and I'm testing the fix. I'll send an update shortly.